packer/scripts/ubuntu/base.sh

```diff
@@ -12,5 +12,3 @@ sudo apt-get update -y
 
 sudo apt-get -y install git curl libcurl3 bridge-utils bundler default-jre-headless unzip wget python-setuptools python-protobuf cgroup-bin ruby1.9.1
 sudo easy_install pip
-
-sudo pip install docker-py==1.1.0
```

Comment:
We need to test this now that https://github.com/Capgemini/Apollo/pull/423 got merged.